dask/dataframe/tests/test_pyarrow.py

```diff
@@ -1,9 +1,14 @@
+from datetime import date, time
+
 import numpy as np
 import pandas as pd
 import pytest
+from _decimal import Decimal
```

Comment:
Why `_decimal` and not `decimal`?

Reply:
This must have gotten In somehow (auto-import maybe?), changed